waku/v2/protocol/waku_store/protocol.nim

```diff
@@ -72,7 +72,16 @@ proc initProtocolHandler(ws: WakuStore) =
     info "received history query", peerId=conn.peerId, requestId=requestId, query=request
     waku_store_queries.inc()
 
-    let responseRes = ws.queryHandler(request)
+    var responseRes: HistoryResult
+    try:
+      responseRes = ws.queryHandler(request)
+    except Exception:
+      error "history query failed", peerId= $conn.peerId, requestId=requestId, error=getCurrentExceptionMsg()
+
+      let response = responseRes.toRPC()
```

Comment:
What would be the content of response here, since `responseRes` raised an exception. Just default values?